admin/lib/solidus_admin/configuration/main_nav.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require "solidus_admin/container"
+require "solidus_admin/main_nav_item"
+
+module SolidusAdmin
+  class Configuration < Spree::Preferences::Configuration
+    # Configuration for the main navigation menu.
+    class MainNav
+      NAMESPACE = "main_nav"
+      private_constant :NAMESPACE
```

Comment:
This is only used in `#container_key` would make sense to just inline it?

Reply:
Yeah, that's an option. If you don't feel strongly about that, I prefer to leave it as it is now. I prefer clarity over terseness for this kind of hardcoded string.

Reply:
whatever, super-weak preference here, I thought it was a leftover 👍